Fastfile

```diff
@@ -150,7 +150,12 @@ def _build_with_gym
   export_method = _get_export_method
   xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
   create_xcconfig(filename: xcconfig_filename)
-  gym(export_method: export_method, xcconfig: xcconfig_filename)
+  gym(configuration: _configuration, export_method: export_method, xcconfig: xcconfig_filename)
+end
+
+def _configuration
```

Comment:
Perhaps this could be renamed `_get_build_config` to keep better consistency with the naming convention elsewhere in this file and to make clearer what config this is.